internal/resource/statefulset_test.go

```diff
@@ -1306,7 +1302,7 @@ var _ = Describe("StatefulSet", func() {
 						},
 						corev1.VolumeMount{
 							Name:      "persistence",
-							MountPath: "/var/lib/rabbitmq/db/",
```

Comment:
FWIW I think `/var/lib/rabbitmq/db/` is a better default. Mnesia will be replaced one day.

Reply:
Ideally we should just mount `/var/lib/rabbitmq` and give r/w permissions to rabbitmq user. Our challenge is that we already mount another volume in `/var/lib/rabbitmq` for the Erlang cookie and IIRC, we can't mount two different volumes in the same path.

Reply:
Going to merge because this is just a fix of the tests and the change you're discussing is already merged. But will add it to our next sync up agenda so we don't lose track